### ISSUE TITLE
Add FixIt section to AAPB homepage

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1177]
**waiting on:**
  - [x] actual comps. This will determine how hard it is.

**Done when**
- [x] We have received comps from Digital for the new homepage section design
- [x] We have implemented the section design comps

**Comment by caseyedavis12** [comment]
![aapb_home_feature](https://cloud.githubusercontent.com/assets/5992135/25042899/b837fa36-20e9-11e7-9fa1-aad5b1d4b0da.jpg)

Here is the section blurb:

You can make historic public radio and television programs from across America easier to search and access. Help AAPB archivists and history lovers identify and correct errors in machine-generated transcripts. Your corrections will made available in the American Archive of Public Broadcasting.

**Comment by caseyedavis12** [comment]
@sroosa, we also need to reorder all the Homepage sections. Feel free to make this a new ticket. Here is the new order:

-Header
-Rotating items with About the AAPB and From the Blog boxes
-Curated Exhibits
-FIX IT
-Browse the Collection
-Participating Organizations
-Every Picture
-Footer

**Comment by caseyedavis12** [comment]
Here are the FIX IT homepage specs!

<img width="1201" alt="aapb_fixit_specs" src="https://cloud.githubusercontent.com/assets/5992135/25064423/a81f91b6-21c7-11e7-8ab1-b21ec4c1517a.png">

**Comment by jasoncorum** [comment]
@sroosa - is the reorder of homepage sections officially part of this ticket?

**Comment by sroosa** [comment]
@jasoncorum it has a separate ticket (#1240), but if they should actually just be one ticket, then we can add that to this one's Done When checklist and delete the other ticket.

**Comment by jasoncorum** [comment]
@sroosa - cool. let's leave as separate for now and depending on how it's all set up, i might roll it in to this one.

**Comment by jasoncorum** [comment]
@sroosa - I mentioned one thing on this in person - that the Fix It logo in the comp is larger than the one we're using in the navigation area. In addition, the logo we received for the navigation area has a small white background which won't work against the Fix It homepage section background color.

Something else I thought of: what should be linked from the comp? I would link the logo to the Fix It homepage (when we get the new one). Then, do want each of the 3 promo boxes (in their entirety) linked to the Fix It homepage? Or just the text in them (Identify Records, Suggest Fixes, and Validate Fixes)?

**Comment by sroosa** [comment]
@jasoncorum I think each of the 3 promo boxes (in their entirety) should be links to the FIX IT homepage too.

**Comment by jasoncorum** [comment]
@sroosa - i ended up doing the work for (#1240) in the same branch, but we can leave as separate tickets since both has some important detailed info. here's a screenshot with both updates made. please let me know if this covers the issues! thanks!

![screencapture-localhost-3000-1493315126616](https://cloud.githubusercontent.com/assets/3814190/25497628/3e534ef4-2b53-11e7-9904-33e5bb6b9d7e.png)

**Comment by sroosa** [comment]
@caseyedavis12 Can you give approval on screenshot of new Fix It Section and ordering?

**Comment by caseyedavis12** [comment]
@sroosa: This looks good to me. The FIX IT logo should link to fixit.americanarchive.org.

**Comment by sroosa** [comment]
@caseyedavis12 The FIX IT logo and  the boxes for each game will all link to fixit.americanarchive.org.

**Comment by caseyedavis12** [comment]
@sroosa: Great!!

**Comment by sroosa** [comment]
@jasoncorum Looks good and covers the issue! Thanks!
